profile/README.md

```diff
@@ -0,0 +1,12 @@
+Welcome to Pulumi open source! :purple_heart: :cloud:
+
+We are glad that you are here! We love and appreciate the Pulumi community.
+We use [Pulumi Slack](https://slack.pulumi.com/)
+and [Pulumi GitHub Discussions](https://github.com/pulumi/pulumi/discussions) for community conversations; use whichever you like.
+
+Want to contribute? Check out the [Pulumi Contributing Guide](https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md).
+
+The [Pulumi CLI and SDK](https://github.com/pulumi/pulumi) are free and open source. If you want Pulumi to manage your state
```

Comment:
Pulumi's infrastructure as code CLI and SDK are free and open source. We also offer the Pulumi Service, a managed service that helps teams and organizations adopt our open source offering at scale and to ship infrastructure faster.

Reply:
thanks George. i am a little bit worried about making this language too marketing focused. but ill leave ur comment unresolved and see what Luke and Joe think is best here, if they support this then I will accept your suggestion.